src/screens/Login/components/Header.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Image, Pressable, StyleSheet, Text, View } from 'react-native';
```

Comment:
Library imports on top, a space and then rest imports

Reply:
Resolved